doc/api/module.md

```diff
@@ -608,10 +608,11 @@ loader with registered `resolve` and `load` hooks; all `require.resolve` calls
 from this module will be processed by the ESM loader with registered `resolve`
 hooks; `require.extensions` and monkey-patching on the CommonJS module loader
 will not apply. (In other words, handled `require` calls will behave similarly
-to an `import` of a CommonJS module.) If `source` is undefined or `null`, it
-will be handled by the CommonJS module loader and `require`/`require.resolve`
-calls will not go through the registered hooks. This behavior for nullish
-`source` is temporary — in the future, nullish `source` will not be supported.
+to an `import` of a CommonJS module, other than `require` being sync.) If
```

Comment:
No, the period goes inside the parentheses when the entire sentence is inside. The period would be outside only if the parenthetical were part of the sentence. https://blog.apastyle.org/apastyle/2013/03/punctuation-junction-periods-and-parentheses.html (see number 2).

Reply:
Wdyt about removing those parentheses?

Reply:
I guess that’s fine. If we do that we should maybe add a paragraph break after it, because it doesn’t really flow with the rest of the paragraph. It’s an aside:

> (In other words, handled `require` calls will behave similarly to an `import` of a CommonJS module, other than `require` being sync.)

Reply:
Want to suggest a specific change?